test/week_calendar_test.exs

```diff
@@ -63,4 +63,24 @@ defmodule Cldr.Calendar.Week.Test do
     assert first_day_of_period == %Date{calendar: Sunday, day: 1, month: 48, year: 2018}
     assert last_day_of_period == %Date{calendar: Sunday, day: 7, month: 53, year: 2018}
   end
+
+  test "regression on Calendar.previous" do
+    defmodule Monday do
+      use Cldr.Calendar.Base.Week, day: 1, month: 1, weeks_in_month: [5, 4, 4], min_days: 7
+    end
+
+    {:ok, today} = Date.convert(~D[2019-04-11], Monday)
+    this_period = Cldr.Calendar.month(today)
+    prior_period_day = Cldr.Calendar.previous(today, :month, coerce: true)
```

Comment:
I don't think the coerce actually matters for the test case, I started with it just in case and forgot to remove.  unsurprisingly I think this one is also rooted around a 53rd week year, but I wasn't sure how to capture the expectation cleanly without dealing in conversions/didn't get a chance to really understand it very well.